docs/en_US/jailbreak/checkra1n/installing-checkra1n-atv/index.md

```diff
@@ -0,0 +1,48 @@
+---
+lang: en_US
+title: Installing checkra1n (Apple TV)
+description: Guide to installing checkra1n for Apple TVs
+permalink: /installing-checkra1n-tv
+pkgman: sileo
+redirect_from:
+  - /ra1n
+  - /installing-odysseyrain
+  - /odysseyra1n
+  - /odysseyrain
+  - /or
+  - /installing-checkra1n
+  - /installing-checkrain
+  - /checkra1n
+  - /checkrain
+  - /cr
```

Comment:
this section should be modified as it will conflict with the existing checkra1n page